tool/lib/model.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+class DevToolsRepo {
+  DevToolsRepo._create(this.repoPath);
+
+  final String repoPath;
+
+  @override
+  String toString() => '[DevTools $repoPath]';
+
+  /// This returns the DevToolsRepo instance based on the current working
+  /// directory.
+  ///
+  /// This can fail and return null if the current working directory is not
+  /// contained within a git checkout of DevTools.
+  static DevToolsRepo getInstance() {
+    final String repoPath = _findRepoRoot(Directory.current);
```

Comment:
remove types throughout this file as well

Reply:
done

We should update the analysis_options.yaml file at the root of the repo with the current style guide for this repo - it'll reduce code review churn.